metricbeat/module/redis/docker-compose.yml

```diff
@@ -2,10 +2,25 @@ version: '2.3'
 
 services:
   redis:
-    image: docker.elastic.co/integrations-ci/beats-redis:${REDIS_VERSION:-3.2.12}-1
+    image: docker.elastic.co/integrations-ci/beats-redis:${REDIS_VERSION:-3.2.12}-${REDIS_DISTRIBUTION:-oss}-1
     build:
       context: ./_meta
+      dockerfile: Dockerfile.${REDIS_DISTRIBUTION:-oss}
       args:
+        REDIS_DISTRIBUTION: ${REDIS_DISTRIBUTION:-oss}
         REDIS_VERSION: ${REDIS_VERSION:-3.2.12}
+    privileged: true
     ports:
       - 6379
+      - 8070
+  redis-enterprise:
+    image: docker.elastic.co/integrations-ci/beats-redis:${REDIS_VERSION:-5.4.10-22}-enterprise-2
+    build:
+      context: ./_meta
+      dockerfile: Dockerfile.enterprise
+      args:
+        REDIS_DISTRIBUTION: enterprise
+        REDIS_VERSION: ${REDIS_VERSION:-5.4.10-22}
+    privileged: true
```

Comment:
Is this added because redis enterprise seems to need `sys_resource`? Could we add only this capability?

Reply:
Fixed.